### PR TITLE
fix(data-table-skeleton): Update hover border color to ui-03

### LINKED ISSUE #1013: DataTableSkeleton highlights row on hover
@scottdickerson commented on [Thu Aug 02 2018](https://github.com/IBM/carbon-components-react/issues/1153)

## Detailed description

> Describe in detail the issue you're having.
The DataTableSkeleton highlights rows if you hover over them with your mouse.  I don't think we want to imply that the Skeleton Table is interactable.
> Is this a feature request (new component, new icon), a bug, or a general issue?
bug.  
> Is this issue related to a specific component?
DataTableSkeleton
> What did you expect to happen? What happened instead? What would you like to see changed?

> What browser are you working in?
Chrome
> What version of the Carbon Design System are you using?
6.18
> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

## Steps to reproduce the issue
Simply hover over the DataTableSkeleton component

https://codesandbox.io/s/zw5p9ynrzl

---

@asudoh commented on [Thu Aug 02 2018](https://github.com/IBM/carbon-components-react/issues/1153#issuecomment-410092127)

@aagonzales @alisonjoseph Any thoughts on this topic...? Thanks!

---

@alisonjoseph commented on [Thu Aug 02 2018](https://github.com/IBM/carbon-components-react/issues/1153#issuecomment-410097740)

This is a bug, it shouldn't have a highlight on hover color. I'm moving this to the carbon-components repo since it is a css issue.

### PR DESCRIPTION
**Pull request by alisonjoseph**
## Overview

Resolves #1013 

### Added

Sample data table skeleton code to example page

## Testing / Reviewing

Check that data table skeleton is not broken

### CODE REVIEW

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thank you for making this quick fix, @alisonjoseph!

File: src/components/skeleton/skeleton.hbs [hunk 1]
```diff
@@ -137,4 +137,7 @@
   <div class="bx--select bx--skeleton">
     <select class="bx--select-input"></select>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+<p>Data Table</p>
+<table class="bx--skeleton bx--data-table-v2"><thead><tr><th></th><th></th><th></th><th></th><th></th></tr></thead><tbody><tr><td><span></span></td><td><span></span></td><td><span></span></td><td><span></span></td><td><span></span></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr></tbody></table>
```
Comment (joshblack) [inline src/components/skeleton/skeleton.hbs]: Would it help in the future to replace this with formatted HTML? Maybe:

```html
<table class="bx--skeleton bx--data-table-v2">
    <thead>
        <tr>
            <th></th>
            <th></th>
            <th></th>
            <th></th>
            <th></th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td><span></span></td>
            <td><span></span></td>
            <td><span></span></td>
            <td><span></span></td>
            <td><span></span></td>
        </tr>
        <tr>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
        </tr>
        <tr>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
        </tr>
        <tr>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
        </tr>
        <tr>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
            <td></td>
        </tr>
    </tbody>
</table>
```

**Review by joshblack — APPROVED** [review]
Awesome! Just an note about the template, totally understand though if it doesn't make it in 👍

File: src/components/skeleton/skeleton.hbs (hunk 1 above)
Comment (alisonjoseph) [reply src/components/skeleton/skeleton.hbs]: Yea, was just being lazy. I can update. 😄

Comment (joshblack) [reply src/components/skeleton/skeleton.hbs]: 😂

**Pull request merged**